### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by Kraker01** [aidan-walden/Twitch-Rewards-Farmer#7]
Hi,

I've set up your script, but I'm experiencing two problems: 
-  Chrome is launching but not moving from twitch' homepage
- the function to get gamesid is not working

I'm using Windows 10.

**Comment by aidan-walden** [comment]
You're right about the first one, will push a commit that fixes this in a few minutes.
Getting game ID's is working on my end, if you could provide more detail about that I would appreciate it

**Comment by Kraker01** [comment]
My fault about game ID, i didn't insert any space between words (BlackDesert instead of Black Desert)

Now I think it's working fine:
PS C:\Twitch-Rewards-Farmer-master> node index.js
Not logged in
(node:8724) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
Saving cookies to C:\Twitch-Rewards-Farmer-master\cookies\cookies-twitch.json
Refreshing stream...
Now watching Salt

(i'm trying to get drops for Black Desert Online)

**Comment by Kraker01** [comment]
Script is starting, but after some time, it hangs on "Refreshing Stream"

I've logged in my Twitch account, under Drops, no campaign is showing Drop progression. The progression only started when I manually selected a drop-enabled streaming

**Comment by aidan-walden** [comment]
Strange. Twitch has been progressively making this type of automation more difficult over time and admittedly I have not been keeping up with it. Last time I checked, there was no easy way to get which streamers had drops enabled programmatically. Are you sure Salt has drops enabled? That could be an explanation, however its very possible that the drops behavior has been updated.
